### ISSUE TITLE
Bias initialized to -inf for class with no examples

### ISSUE THREAD

**Issue by kevin-qi** [jbohnslav/deepethogram#15]
One of my classes has 0 samples and the bias for it was initialized to -inf, leading to NaN training loss. I was able to fix this by making a small change in `feature_extractor/models/CNN.py`

from
```python
bias = torch.log(torch.from_numpy(pos / neg)).float()
bias = torch.nn.Parameter(bias)
linear_layer.bias = bias
log.info('Custom bias: {}'.format(linear_layer.bias))
```
to
```python
bias = np.nan_to_num(np.log(pos / neg), neginf = 0.0001) # Might want a better default value here
bias = torch.nn.Parameter(torch.from_numpy(bias).float())
linear_layer.bias = bias
log.info('Custom bias: {}'.format(linear_layer.bias))
```

Just wanted to raise this issue in case this is not intended behavior for classes with no examples!

**Comment by jbohnslav** [comment]
Thanks for the catch, and the quick fix! Definitely not the intended behavior. I set bias to be 0, which it would be if not for this custom initialization. Fixed with a956a42
